### ISSUE TITLE
Filling cycleway quest answer by tapping apply previous answer button makes `oneway:bicycle=no` option disappear

### ISSUE THREAD

**Issue by wielandb** [streetcomplete/StreetComplete#4744]
**How to Reproduce**

- Have previously applied `cycleway:both=no` to a road so that the previous answer-button shows two red X's
- Open cycleway quest on a oneway road
- Tap the previous answer button ("none" and "none, cycling not allowed here" will be filled out)
- Try to change "none, cycling not allowed here" to "none, but cyclists may use oneway road in opposite direction"
- It's not possible because the option is not available
- Go out of the quest, go back in and manually enter the cycleways, the option is now available

https://user-images.githubusercontent.com/29458682/211687346-d9752e46-b4d3-4168-bb17-fe1b9a6b41aa.mp4

**Expected Behavior**
- It is possible to change the selection to "none, but cyclists may use road in opposite direction", especially after the user is promted to double-check if this may be the case

**Versions affected**
Android 10, StreetComplete 50.2

**Comment by westnordost** [comment]
Interesting also that the pictures are displayed upside-down.

**Comment by westnordost** [comment]
There is something fishy. Why is the cycleway quest displayed at all while the cycleway overlay is active? It should be hidden. Also, why is the cycleway overlay form shown when you click on the cycleway quest icon?

**Comment by westnordost** [comment]
Hm I cannot reproduce this.

https://user-images.githubusercontent.com/4661658/214338369-05117751-0921-48d4-a494-8944887c7f5a.mp4

**Comment by westnordost** [comment]
Are the icons always displayed upside-down for you? This is definitely wrong and the only way I think this can happen is if the app thinks you are in England or somewhere else where people drive on the left side.

**Comment by westnordost** [comment]
No actually, in England the icons look like this

![image](https://user-images.githubusercontent.com/4661658/214339636-581045af-617e-4b9b-939f-d054c5725a14.png)

**Comment by westnordost** [comment]
Are you using some modified version of StreetComplete?

**Comment by westnordost** [comment]
Please check if it is really v50.2 you are using. That the cycleway labels are displayed in the same orientation as the streets was changed for v50.2 and specifically #4722 was fixed for v50.2 and what you showed may actually be another side-effect of #4722.

**Comment by wielandb** [comment]
> Please check if it is really v50.2 you are using. That the cycleway labels are displayed in the same orientation as the streets was changed for v50.2 and specifically #4722 was fixed for v50.2 and what you showed may actually be another side-effect of #4722.

It turns out that I [was indeed using 50.1](https://www.openstreetmap.org/changeset/130928126) when I captured the video on Jan 6th. When I reported the bug five days later on Jan 11th, my phone had indeed updated to version 50.2 in the meantime and so when I checked on my phone which version I was using when reporting the bug it showed 50.2 and I did not think anything of it. Sorry. 🙇‍♂️

> Are you using some modified version of StreetComplete?

No, I am using the APKs from Github/Google Play

> Are the icons always displayed upside-down for you? This is definitely wrong and the only way I think this can happen is if the app thinks you are in England or somewhere else where people drive on the left side.

To be sure, I tested it again with 50.1 and 50.2. And yes, the icons are upside down on _some, but not all_ ways in 50.1
The two ther ways tested in the video are https://www.openstreetmap.org/way/164759906 and https://www.openstreetmap.org/way/14101811 . I can't see anything in the tags of theese two ways that explains the difference in behaviour. But as also visible in the video, updating to 50.2 solves all issues, so I guess you're right and it was a side-effect of #4722 and is fixed now. 

https://user-images.githubusercontent.com/29458682/214982586-479b96e0-44b1-4f43-9288-5641ed391a98.mp4
